installation/configuration.php-dist

```diff
@@ -36,6 +36,7 @@ class JConfig
 	public $list_limit = 20;
 	public $access = 1;
 	public $frontediting = 1;
+	public $frontinlineediting = true;
```

Comment:
@rs4231199 This default value should be false as it is also false in your XML file. The default value of false is the correct one.